compiler/rustc_ast/src/ast.rs

```diff
@@ -1095,7 +1095,11 @@ impl Expr {
     pub fn is_potential_trivial_const_param(&self) -> bool {
         let this = if let ExprKind::Block(ref block, None) = self.kind {
             if block.stmts.len() == 1 {
-                if let StmtKind::Expr(ref expr) = block.stmts[0].kind { expr } else { self }
+                if let StmtKind::Expr(ref expr) = block.stmts[0].kind {
+                    expr
+                } else {
+                    self
+                }
```

Comment:
https://github.com/rust-lang/rust/pull/78978#discussion_r523200125
CI will probably fail due to this.